csfieldguide/templates/interactives/rgb-mixer.html

```diff
@@ -4,11 +4,14 @@
 {% load static %}
 
 {% block html %}
-  <div id="interactive-rgb-mixer" class="mb-3 container-fluid">
+  <div id="interactive-rgb-mixer" class="my-3 container-fluid">
+    <img id="pixelmania-logo" src="{% static 'interactives/rgb-mixer/img/pixelmania-logo-half-size.png' %}" class="d-none img-fluid" crossorigin="anonymous">
```

Comment:
Note for future: we should put this picture in a shared folder somewhere so we don't need three slightly different copies